style.css

```diff
@@ -1,391 +1,568 @@
+    margin-left: 1em;
+    margin-right: 1em;
   }
 
-  .header-container {
-    margin-top: 4em;
+  .all-skills * {
+    width: auto;
   }
+}
 
-  .code-skill {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: center;
-    width: 80%;
+@media only screen and (min-width: 700px) {
+  #home-paragraph {
+    width: 53.65%;
   }
 
-  .lang {
-    align-self: center;
-    font-size: small;
-    margin-left: 0;
-    margin-right: 5px;
-    margin-bottom: 10px;
-    padding-left: 2px;
-    padding-right: 2px;
-    padding-top: 5px;
-    padding-bottom: 5px;
-    border-radius: 8px;
-    background-color: #fff;
-    color: #36b37e;
+  .aside-container {
+    background: url('images/illustration-about-me-2-desktop.svg') no-repeat, url('images/illustration-about-me-desktop.svg') no-repeat;
+    background-position: left bottom, right;
   }
 
-  #code-lang-list {
-    justify-content: center;
-    min-width: 100%;
+  .header-container-about {
+    margin-top: 1em;
+    background: none;
   }
 
-  #frame-wrk-list {
-    min-width: 70%;
+  .about-paragraph {
+    width: 53.65%;
   }
 
-  .extra {
-    min-width: 100%;
-    padding: 0;
+  .form-sec {
+    max-width: 80%;
+    flex-direction: row;
+    align-items: center;
+    margin-bottom: 15em;
   }
 
   .last-head-container {
-    text-align: center;
-    display: flex;
-    justify-content: center;
+    text-align: initial;
+    width: 50%;
+    padding-right: 2em;
+    justify-content: stretch;
+    align-items: stretch;
   }
 
-  .header-3 {
-    font-size: 40px;
-    color: #172b4d;
-    margin-left: 0.5em;
-    margin-right: 0.5em;
+  .last-head-container > * {
+    font-size: 30px;
   }
 
   .form {
-    display: flex;
-    flex-direction: column;
-    margin: 1em;
+    width: 50%;
+    margin-right: 0;
+    margin-top: 3em;
+    border: 2px;
   }
 
-  #name {
+  .names {
     display: flex;
-    flex-direction: column;
-    margin-bottom: 1em;
+    margin-bottom: 0.5em;
   }
 
-  input {
-    padding: 0.7em;
+  .names > input:nth-of-type(1) {
+    margin-right: 0.5em;
   }
 
   #get-in-touch {
-    margin-left: 40%;
-    margin-top: 1em;
+    margin-left: 0;
   }
 
-  .footer * {
-    justify-content: center;
+  .skillset {
+    background:
+      url('images/box.svg') no-repeat,
+      url('images/illustration-2-contact-form-desktop.svg ') no-repeat,
+      url('images/contact-form-illustratoin-3-desktop.svg') no-repeat;
+    background-position: bottom left, 20% 100%, right 100%;
   }
+}
 
-  #code-lang-list * {
-    min-width: 20%;
+@media only screen and (min-width: 900px) {
+  body {
+    background: url('images/Header-llustration-desktop.svg');
+    background-repeat: no-repeat;
+    background-size: 1200px;
   }
 
-  .mail-container {
-    display: flex;
+  .cards {
+    width: fit-content;
+    grid-template-columns: repeat(3, 1fr);
+    grid-column-gap: 1em;
   }
 
-  .mail {
-    color: #42526e;
-    text-decoration: none;
+  .sample-containers {
+    min-width: 20em;
```

Comment:
- [ ] Although they are placeholders, it would be better to apply a `min-width` that allows each card to be wide enough for its contents as in the template and use the given fonts for this section.
Kindly have a look at the `width` of a card on desktop from the template and set it as the `min-width` for your cards. You can use this [px to em converter](https://www.w3schools.com/tags/ref_pxtoemconversion.asp). Also, use [Poppins](https://fonts.google.com/specimen/Poppins?sidebar.open=true&selection.family=Amatic+SC:wght@700&query=popp) for the `h3` tags and [Inter](https://fonts.google.com/specimen/Inter?sidebar.open=true&selection.family=Amatic+SC:wght@700&query=Inter) for the other elements